ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+NOTE: ISSUES ARE NOT FOR CODE HELP - Ask for Help at https://stackoverflow.com
+
+Your issue may already be reported!
+Also, please search on the [issue tracker](../) before creating one.
+
+* **I'm submitting a ...**
+  - [ ] bug report
+  - [ ] feature request
+  - [ ] support request => Please do not submit support request here, see note at the top of this template.
+
+# Issue Description
+* When Issue Happens
+* Steps To Reproduce
+  1.
+  1.
+  1.
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Your Code
+
+```
+If relevant, paste all of your challenge code here
+```
+
+## Error message
+
+```
+If relevant, paste all of your error messages here
+```
+
+## Your Local environment
+* Operating System, version
+* Python, version
+
```

Comment:
maybe we should also ask for pip freeze as it will be useful to reproduce in case its a bug